adoc/transfer.1.adoc

```diff
@@ -178,4 +178,28 @@ godata mygodatadir -r
 ----
 
 
+Consume a batch of files to transfer from a data file, submit the transfer
+task, get back its task ID for use in `globus task wait`, wait for up to 30
+seconds for the task to complete, and then print a success or failure message.
+
+----
+$ source_ep=ddb59aef-6d04-11e5-ba46-22000b92c6ec
+$ dest_ep=ddb59af0-6d04-11e5-ba46-22000b92c6ec
+$ cat my_file_batch.txt
+/share/godata/file1.txt ~/myfile1.txt
+/share/godata/file2.txt ~/myfile2.txt
+/share/godata/file3.txt ~/myfile3.txt
+$ task_id="$(globus transfer $source_ep $dest_ep \
+    --jmespath 'task_id' --format=UNIX \
+    --batch < my_file_batch.txt)"
+$ echo "Waiting on 'globus transfer' task '$task_id'"
+$ globus task wait "$task_id" --timeout 30
+$ if [ $? -eq 0 ]; then
+    echo "$task_id completed successfully";
+  else
+    echo "$task_id failed!";
+  fi
```

Comment:
You switch back and forth in this PR between presenting examples as if they were interactive (with leading `$`s for each command) and as if they were in a script (without the leading `$`s).

I think this example is definitely a candidate for the "as if they were in a script" treatment if not for the `cat`ing of `my_file_batch.txt` to show what is in it.

Maybe if you did want to show this one as if it were a script, you could either have a separate (labeled) AsciiDoc code block for `my_file_batch.txt` or you could use a `<<END` heredoc in the script.

Reply:
After hemming and hawing a bit, I think I'm just going to write two separate code blocks. One that reads as `$ cat my_file_batch.txt` and a following one without the leading `$`.

I want to avoid the heredoc for people who won't know what it is.